### ISSUE TITLE
Inconsistant encoding and results

### ISSUE THREAD

**Issue by risseraka** [livingdata-co/addok-geocode-stream#14]
http://api-adresse.data.gouv.fr/search/?q=18%20boulevard%20Laromigui%C3%A8re%2012000%20RODEZ

vs.

```bash
echo 'adresse\n18 Boulevard Laromiguière 12000 Rodez' | ./cli.js

# adresse,latitude,longitude,result_label,result_score,result_type,result_id,result_housenumber,result_name,result_street,result_postcode,result_city,result_context,result_citycode,result_oldcitycode,result_oldcity,result_district
# 18 Boulevard Laromiguière 12000 Rodez,44.351057,2.577754,18 Boulevard de la R�publique 12000 Rodez,0.61,housenumber,12202_1580_00018,18,Boulevard de la R�publique,,12000,Rodez,"12, Aveyron, Occitanie",12202,,,


echo 'adresse,postcode\n18 Boulevard Laromiguière 12000 Rodez,12000' | ./cli.js --postcode postcode --columns adresse

# adresse,postcode,latitude,longitude,result_label,result_score,result_type,result_id,result_housenumber,result_name,result_street,result_postcode,result_city,result_context,result_citycode,result_oldcitycode,result_oldcity,result_district
# 18 Boulevard LaromiguiÃ¨re 12000 Rodez,12000,44.351057,2.577754,18 Boulevard de la République 12000 Rodez,0.61,housenumber,12202_1580_00018,18,Boulevard de la République,,12000,Rodez,"12, Aveyron, Occitanie",12202,,,

```

**Comment by jdesboeufs** [comment]
Fixed since `0.12`. Feel free to re-open if necessary :)

**Comment by risseraka** [comment]
LGTM!
